electron install failed via  command "npm run package-darwin"

> AOMAnalyzer@1.0.0 package-darwin /Users/edward.shi/project/AOM/aomanalyzer
> electron-packager . --platform=darwin --arch=x64 --electron-version=1.4.16 --overwrite --icon=vomit.icns --out=release_builds --tmpdir=./

Packaging app for platform darwin x64 using electron v1.4.16
ENAMETOOLONG: name too long, unlink '/Users/edward.shi/project/AOM/aomanalyzer/electron-packager/darwin-x64/AOMAnalyzer-darwin-x64/Electron.app/Contents/Resources/app/electron-packager/darwin-x64/AOMAnalyzer-darwin-x64/Electron.app/Contents/Resources/app/electron-packager/darwin-x64/AOMAnalyzer-darwin-x64/Electron.app/Contents/Resources/app/electron-packager/darwin-x64/AOMAnalyzer-darwin-x64/Electron.app/Contents/Resources/app/electron-packager/darwin-x64/AOMAnalyzer-darwin-x64/Electron.app/Contents/Resources/app/electron-packager/darwin-x64/AOMAnalyzer-darwin-x64/Electron.app/Contents/Resources/app/electron-packager/darwin-x64/AOMAnalyzer-darwin-x64/Electron.app/Contents/Resources/app/electron-packager/darwin-x64/AOMAnalyzer-darwin-x64/Electron.app/Contents/Resources/app/electron-packager/darwin-x64/AOMAnalyzer-darwin-x64/Electron.app/Contents/Resources/app/electron-packager/darwin-x64/AOMAnalyzer-darwin-x64/Electron.app/Contents/Resources/app/node_modules/ts-loader/examples/react-babel-karma-gulp-thread-loader/test/components/Greeting.tests.tsx'
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! AOMAnalyzer@1.0.0 package-darwin: `electron-packager . --platform=darwin --arch=x64 --electron-version=1.4.16 --overwrite --icon=vomit.icns --out=release_builds --tmpdir=./`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the AOMAnalyzer@1.0.0 package-darwin script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/edward.shi/.npm/_logs/2018-02-08T10_40_24_563Z-debug.log

--- a/package.json
+++ b/package.json
@@ -8,9 +8,9 @@
     "build-watch": "webpack --watch --config config/webpack.config.js",
     "build-debug": "webpack --display-reasons --progress --config config/webpack.config.js",
     "build-release": "webpack -p --config config/webpack.config.js",
-    "package-darwin": "electron-packager . --platform=darwin --arch=x64 --electron-version=1.4.16 --overwrite --icon=vomit.icns --out=release_builds --tmpdir=./",
-    "package-linux": "electron-packager . --platform=linux --arch=x64 --electron-version=1.4.16 --overwrite  --out=release_builds --tmpdir=./",
-    "package-win32": "electron-packager . --platform=win32 --arch=x64 --electron-version=1.4.16 --overwrite  --out=release_builds --tmpdir=./"
+    "package-darwin": "electron-packager . --platform=darwin --arch=x64 --electron-version=1.4.16 --overwrite --icon=vomit.icns --out=release_builds --tmpdir=/tmp",
+    "package-linux": "electron-packager . --platform=linux --arch=x64 --electron-version=1.4.16 --overwrite  --out=release_builds --tmpdir=/tmp",
+    "package-win32": "electron-packager . --platform=win32 --arch=x64 --electron-version=1.4.16 --overwrite  --out=release_builds --tmpdir=/tmp"
   },
   "author": "Michael Bebenita",
   "license": "ISC",

